### ISSUE TITLE
Linear regressions not working with sklearn 1.2 due to `normalize` argument removal

### ISSUE THREAD

**Issue by dvasya** [alan-turing-institute/MLJScikitLearnInterface.jl#45]
Seems like linear-regressors.jl (and possibly elsewhere) needs cleanup to make it work with the latest sklearn (it was producing deprecation warnings before but as of 1.2 just doesn't work anymore).

https://github.com/scikit-learn/scikit-learn/pull/17743

Thanks!

**Comment by OkonSamuel** [comment]
Thanks @dvasya for reporting this issue.

**Comment by ablaom** [comment]
Closed by #50. Thanks @dvasya for your patience.
